webpack/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var StringReplacePlugin = require('string-replace-webpack-plugin');
+
+var proxy;
+
+try {
+  // If this fails - you should create a proxy.dev.js by copying proxy.template.js
+  // and customizing the proxy as needed.
+  proxy = require('./proxy.dev.js');
+} catch (err) {
+  proxy = require('./proxy.template.js');
+}
+
+module.exports = require('./make-webpack-config')({
```

Comment:
I'm not that sure about this `make-webpack-config` process. It seems a little over engineered and is hard to follow. Why not one very explicit and descriptive configuration file for the default configuration and a second one that extends/overwrites (`Object.assign`) specific configurations?